ar4_hardware_interface/ar4_firmware/teensy_arm.ino

```diff
@@ -0,0 +1,412 @@
+#include <AccelStepper.h>
```

Comment:
Do we know the source of this file? Just to check if we need to add a header

Reply:
It is an arduino library. It is not necessary to add the heather.